plugins/org.python.pydev.core/helpers/load-conda-vars.bat

```diff
@@ -10,13 +10,15 @@
 
 @set "CONDA_PREFIX=%__PYDEV_CONDA_PREFIX__%"
 @set "CONDA_DEFAULT_ENV=%__PYDEV_CONDA_DEFAULT_ENV__%"
+@set "CONDA_ACTIVATION=%__PYDEV_CONDA_ACTIVATION__%"
 @set "CONDA_ENV_PATH=%CONDA_PREFIX%"
 @set "CONDA_PATH_BACKUP=%PATH%"
 @set "PATH=%CONDA_PREFIX%;%CONDA_PREFIX%\Scripts;%CONDA_PREFIX%\Library\bin;%PATH%"
 
 @REM activate conda and get the environment
-@if exist "%CONDA_PREFIX%" (
-	@call do conda activate %CONDA_PREFIX%
+@if exist "%CONDA_ACTIVATION%" (
```

Comment:
If don't think we need the if exist, just call it and let it fail if it's not there (we already check it in the java side anyways).